src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -107,32 +109,25 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
-register_cli_argument('appservice web config backup create', 'database_name', help='Name of the database to backup')
-register_cli_argument('appservice web config backup create', 'database_connection_string', help='Connection string for the database to backup')
-register_cli_argument('appservice web config backup create', 'database_type', help='Type of database being backed up', **enum_choice_list(database_types))
-
-register_cli_argument('appservice web config backup update', 'storage_account_url', help='SAS URL for the blob storage container')
-register_cli_argument('appservice web config backup update', 'schedule_frequency', help='Numerical frequency for how often to backup')
-register_cli_argument('appservice web config backup update', 'schedule_frequency_unit', help='Unit for how often to backup', **enum_choice_list(frequency_units))
-register_cli_argument('appservice web config backup update', 'keep_at_least_one_backup', help='If true, one backup will always be retained in your storage container', **enum_choice_list(two_states_switch))
-register_cli_argument('appservice web config backup update', 'retention_period_in_days', help='How long to keep a backup before automatically deleting it. Set to 0 for indefinite retention')
-register_cli_argument('appservice web config backup update', 'database_name', help='Name of the database to backup')
-register_cli_argument('appservice web config backup update', 'database_connection_string', help='Connection string for the database to backup')
-register_cli_argument('appservice web config backup update', 'database_type', help='Type of database being backed up', **enum_choice_list(database_types))
-
-register_cli_argument('appservice web config backup restore', 'storage_account_url', help='SAS URL for the blob storage container')
-register_cli_argument('appservice web config backup restore', 'storage_blob_name', help='Name of the backup blob to restore (e.g. backup.zip)')
-register_cli_argument('appservice web config backup restore', 'database_name', help='Name of the database to restore')
-register_cli_argument('appservice web config backup restore', 'database_type', help='Type of database to restore', **enum_choice_list(database_types))
-register_cli_argument('appservice web config backup restore', 'database_connection_string', help='Connection string of database server for the restored database')
-register_cli_argument('appservice web config backup restore', 'target_name', help='Name of the app which will be overwritten with the restored data')
-register_cli_argument('appservice web config backup restore', 'overwrite', help='Must be true to allow the source web app to be overwritten')
-register_cli_argument('appservice web config backup restore', 'ignore_hostname_conflict', help='If true, ignores custom hostnames stored in the backup')
+db_types = [DatabaseServerType.my_sql, DatabaseServerType.sql_azure]
+frequency_units = [FrequencyUnit.hour, FrequencyUnit.day]
```

Comment:
I'll have to keep the databases in their own list since we don't support backing up all the ones listed in the enum, but I can use FrequencyUnit as you suggested